src/midiEditorCommands.cpp

```diff
@@ -2292,6 +2293,96 @@ void postToggleFunctionKeysAsStepInput(int command) {
 	}
 }
 
+void postMidiToggleMute(int command) {
+	HWND editor = MIDIEditor_GetActive();
+	MediaItem_Take* take = MIDIEditor_GetTake(editor);
+	int eventCount = countSelectedEvents(take);
+	if (eventCount == 0) {
+		return;
+	}
+	// Get selected notes.
+	vector<MidiNote> selectedNotes = getSelectedNotes(take);
+	// Get selected CCs.
+	vector<MidiControlChange> selectedCCs = getSelectedCCs(take);
+	int noteCount = selectedNotes.size();
+	int CCCount = selectedCCs.size();
+	ostringstream s;
+	if (CCCount == 0) { // If only notes are selected
+		fakeFocus = FOCUS_NOTE;
+		if (settings::reportNotes) {
+			if (noteCount == 1) {
+				if (selectedNotes[0].muted) {
+					s << translate("muted") << " ";
+				} else {
+					s << translate("unmuted") << " ";
+				}
+				s << getMidiNoteName(take, selectedNotes[0].pitch, selectedNotes[0].channel);
+			} else {
+				int mutedCount = count_if(selectedNotes.begin(), selectedNotes.end(), [](auto note) { return note.muted; });
+				int unmutedCount = noteCount - mutedCount;
+				if (mutedCount > 0) {
+					// Translators: used when reporting the number of muted notes.
+					// {} will be replaced by the number of muted notes. E.g. "3 notes muted"
+					s << " " << format(
+					translate_plural("{} note muted", "{} notes muted", mutedCount), mutedCount);
```

Comment:
When wrapping code onto a new line like this, please indent an additional level so it's clear that it's a continuation of the previous line.